.clang-format

```diff
@@ -0,0 +1,15 @@
+# Common settings
```

Comment:
Hi @triccyx 
Do you think we can restrain the rules to the smallest set? For example, to start with only rules about tabs and identwidht?

All the rest is too strict and conflicts with the developers' flavors.

Reply:
@triccyx please respond to the correct message.
I'm gonna delete this.